src/engines/misc/external/external_in.hpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
```

Comment:
nit: unify filenames: `external/external.hpp` or `external_in/external_in.hpp`

Reply:
I just didn't want to confuse this with the folder for external libraries. But I can easily change it.